From a65e8be0dbde0e23fed88f8ebdcdf8668b238b9c Mon Sep 17 00:00:00 2001
From: Ali Mohammad Pur <ali.mpfard@gmail.com>
Date: Wed, 12 Jan 2022 22:18:02 +0330
Subject: [PATCH 4/6] cmcurl: Include unistd

Everyone gets this wrong. Most platforms are very lax with these includes, but we're not one of them.

- [X] Local?
- [ ] Should be merged to upstream?
- [ ] Resolves issue(s) with our side of things
- [ ] Hack
---
 Utilities/cmcurl/include/curl/multi.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Utilities/cmcurl/include/curl/multi.h b/Utilities/cmcurl/include/curl/multi.h
index 91cd95d3..9940748f 100644
--- a/Utilities/cmcurl/include/curl/multi.h
+++ b/Utilities/cmcurl/include/curl/multi.h
@@ -47,6 +47,7 @@
  * but with this warning attached.
  */
 #include "curl.h"
+#include <unistd.h>
 
 #ifdef  __cplusplus
 extern "C" {
-- 
2.36.1

